assets/scss/3-components/1-carousel.scss

```diff
@@ -45,19 +54,81 @@
   align-items: center;
   justify-content: center;
   position: relative;
-  z-index: 1;
+  z-index: 2;
   text-align: center;
 
   span {
     font-size: 2.4em;
     font-family: $font-secondary;
     letter-spacing: 1px;
+    opacity: 0;
+    transform: translateY(20px);
+    transition: transform 0.2s $cubic, opacity 0.2s $cubic;
+
+    .carousel-list__item--active & {
+      opacity: 1;
+      transform: translateY(0px);
+      transition: transform 0.5s $cubic 200ms, opacity 0.5s $cubic 200ms;
```

Comment:
Consider if this is correct. This will transpile to:
```css
.carousel-list__info span .carousel-list__item--active .carousel-list__info span { ... }
```

If `@at-root` is used

```scss
@at-root .carousel-list__item--active & { ... }
```

Then you'd have
```css
.carousel-list__item--active .carousel-list__info span { ... }
```